applications/fishing-map/src/features/workspace/workspace.selectors.ts

```diff
@@ -197,16 +197,20 @@ export const selectDataviewInstancesResolved = createSelector(
     // resolved array filters to url filters
     dataviewInstancesResolved = dataviewInstancesResolved.map((dataviewInstance) => {
       if (dataviewInstance.config?.type === Generators.Type.HeatmapAnimated) {
-        const dataviewInstanceWithUrlFilter = {
-          ...dataviewInstance,
-        }
-        if (dataviewInstance.config.filters?.length && dataviewInstanceWithUrlFilter.config) {
-          const flags = dataviewInstanceWithUrlFilter.config.filters
-          dataviewInstanceWithUrlFilter.config.filter = `flag IN (${flags
-            .map((f: string) => `'${f}'`)
-            .join(', ')})`
+        const { filters } = dataviewInstance.config
```

Comment:
I think you could set the default value for `filters` here so that you don't need the if cause later.
```ts
const { filters = {} } = dataviewInstance.config
```

Reply:
I like the approach but in this particular case doing it means I'll need later to check if it is an empty array to avoid doing
```
Object.keys(filters)...
```
so does it really change something ?

Reply:
It's just a minor improvement, it does not change the behavior. Note that if `filters` receives its default value `{}` (empty object) then it's not necessary to check anything since `Object.keys(filters)` will be an empty array and thus `sqlFilters` too.